CONTRIBUTING.md

```diff
@@ -0,0 +1,55 @@
+# Contributing to Better Evidence Project
+
+## Project Overview
+
```

Comment:
i'm not really sure if you want to keep this, this PR is not really for working on any documentation and i dont want to givw feedback on this